ambry-store/src/test/java/com.github.ambry.store/IndexTest.java

```diff
@@ -782,6 +782,36 @@ public void findEntriesSinceOnRestartTest() throws IOException, StoreException {
   }
 
   /**
+   * Tests {@link PersistentIndex#getLogSegmentsNotInJournal()}
+   * @throws InterruptedException
+   * @throws StoreException
+   * @throws IOException
+   */
+  @Test
+  public void getLogSegmentsNotInJournalTest() throws InterruptedException, IOException, StoreException {
+    if (isLogSegmented) {
+      testGetLogSegmentsNotInJournal(2, 2 * MAX_IN_MEM_ELEMENTS, false);
+      testGetLogSegmentsNotInJournal(3, 2 * MAX_IN_MEM_ELEMENTS, false);
```

Comment:
what is the intent of this function call? `testGetLogSegmentsNotInJournal()` seems to reduce this to the case tested in the previous function call.